carthage_command.sh

```diff
@@ -0,0 +1,14 @@
+xcodevers=`xcodebuild -scheme Focus  -showBuildSettings  | grep -i 'SDK_VERSION =' | sed 's/[ ]*SDK_VERSION = //' | awk NF=1 FPAT=..`
```

Comment:
`| awk NF=1 FPAT=..` fails on the builders, not sure why as it works locally, use `| colrm 3` instead